File: java/program/Product.java

public class Product {
    protected int idProduct;
    protected String name;
    protected String brand;
    protected double price;

    // Konstruktor
    public Product(int idProduct, String name, String brand, double price) {
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    public int getId() {
        return idProduct;
    }

    public String getName() {
        return name;
    }

    public String getBrand() {
        return brand;
    }

    public double getPrice() {
        return price;
    }
}

File: python/program/Product.py
class Product:
    def __init__(self, idProduct, name, brand, price):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price

    def getId(self):
        return self.idProduct

    def getName(self):
        return self.name

    def getBrand(self):
        return self.brand

    def getPrice(self):
        return self.price
